src/Model/Actions.re

```diff
@@ -122,10 +122,7 @@ type t =
   | CopyActiveFilepathToClipboard
   | SearchStart
   | SearchHotkey
-  | SearchInput(string)
-  | SearchInputClicked(int)
-  | SearchUpdate([@opaque] list(Ripgrep.Match.t))
-  | SearchComplete
+  | Search(Feature_Search.msg)
```

Comment:
Excellent - your guidance to extract the 'Actions' out this way will help us support additional 'feature extractions' 👍